server/src/main/java/org/apache/uniffle/server/LocalStorageChecker.java

```diff
@@ -73,7 +73,8 @@ public boolean checkIsHealthy() {
     int corruptedDirs = 0;
 
     for (StorageInfo storageInfo : storageInfos) {
-      if (!storageInfo.checkStorageReadAndWrite()) {
+      if (storageInfo.checkIsSpaceEnough()
```

Comment:
`checkIsSpaceEnough` can't guaratee that the disk is full. It only means that the disk capacity is very high.